client/src/main/java/pl/edu/agh/ClientCallback.java

```diff
@@ -0,0 +1,11 @@
+package pl.edu.agh;
+
+/**
+ * Created by Andrzej on 2016-11-11.
```

Comment:
You want to remove package?? Without it it won't compile ;) Or do you want to move ClientCallback out of pl.edu.agh package?